github/codesofconduct.go

```diff
@@ -0,0 +1,81 @@
+// Copyright 2023 The go-github AUTHORS. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package github
+
+import (
+	"context"
+	"fmt"
+)
+
+// CodesOfConductService provides access to code-of-conduct-related functions in the GitHub API.
+type CodesOfConductService service
+
+// CodeOfConduct represents a code of conduct.
+type CodeOfConduct struct {
+	Name *string `json:"name,omitempty"`
+	Key  *string `json:"key,omitempty"`
+	URL  *string `json:"url,omitempty"`
+	Body *string `json:"body,omitempty"`
+}
+
+func (c *CodeOfConduct) String() string {
+	return Stringify(c)
+}
+
+// ListCodesOfConduct returns all codes of conduct.
+//
+// GitHub API docs: https://docs.github.com/rest/codes-of-conduct/codes-of-conduct#get-all-codes-of-conduct
+func (s *CodesOfConductService) ListCodesOfConduct(ctx context.Context) ([]*CodeOfConduct, *Response, error) {
```

Comment:
Another thought for you to consider...

Since we are breaking the API anyway with this change (because of `Markdown`), what do you think about reducing the stutter here by renaming `CodesOfConductService.ListCodesOfConduct` to `CodesOfConductService.List`, and same for `Get`, etc?

I looked through this repo to see if there is precedent for this, and there does appear to be:

```bash
$ rg ' List\('
github/codespaces.go
128:func (s *CodespacesService) List(ctx context.Context, opts *ListCodespacesOptions) (*ListCodespaces, *Response, error) {

github/licenses.go
64:func (s *LicensesService) List(ctx context.Context) ([]*License, *Response, error) {

github/gists.go
101:func (s *GistsService) List(ctx context.Context, user string, opts *GistListOptions) ([]*Gist, *Response, error) {

github/gitignore.go
32:func (s *GitignoresService) List(ctx context.Context) ([]string, *Response, error) {

github/pulls.go
146:func (s *PullRequestsService) List(ctx context.Context, owner string, repo string, opts *PullRequestListOptions) ([]*PullRequest, *Response, error) {

github/repos.go
269:func (s *RepositoriesService) List(ctx context.Context, user string, opts *RepositoryListOptions) ([]*Repository, *Response, error) {

github/orgs.go
176:func (s *OrganizationsService) List(ctx context.Context, user string, opts *ListOptions) ([]*Organization, *Response, error) {

github/issues.go
137:func (s *IssuesService) List(ctx context.Context, all bool, opts *IssueListOptions) ([]*Issue, *Response, error) {
```